scripts/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "devDependencies": {
     "@react-native-community/cli": "^7.0.3",
-    "@rnx-kit/jest-preset": "^0.1.9",
+    "@rnx-kit/jest-preset": "^0.1.12",
```

Comment:
Wasn't sure if this was the right way to update the package? Usually I see dependabot update just the yarn.lock file. Should I wait for the dependabot PR to update merge it?

Reply:
I think this is right. This is setting the minimum, which we need because that's the minimum version that has the fix to support win32 specific tests.